tools/github-actions/release/action.yml

```diff
@@ -7,9 +7,17 @@ inputs:
   target:
     description: 'The branch to target'
     required: true
+  previousVersionsLimit:
+    description: 'Limit of previous versions to consider for the release notes'
+    required: false
+    default: '100'
 runs:
   using: "composite"
   steps:
+    - name: Previous release tag
+      if:  ${{!contains( inputs.version, '-' )) && endsWith( inputs.version, '0' )}}
```

Comment:
How does this work for versions ending in `-prerelease.0`?

Reply:
the `!contains( inputs.version, '-' ) ` will exclude it